lib/new-admin/routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const router = express.Router()
+
+const getUserData = function (req, res, next) {
+  const lidCookie = req.cookies && req.cookies.lid
+  if (!lidCookie) {
+    res.sendStatus(403)
+    return
+  }
+
+  const user = req.session.user
+  return res.status(200).json({ message: 'Success', user: user })
+}
```

Comment:
Don't we already have this function on another file? Also why is this outside of graphql?

Reply:
If I recall correctly, this 'duplicate' was due to the react app needing to request this user data, but can't do it through apollo